src/main.rs

```diff
@@ -1,10 +1,13 @@
-// #![warn(clippy::pedantic)]
-// #![allow(clippy::uninlined_format_args)]
-// #![allow(clippy::enum_glob_use)]
-// #![allow(clippy::module_name_repetitions)]
-// #![allow(clippy::doc_markdown)]
+// Temporarily uncomment to view more pedantic or new nursery lints.
+// #![warn(clippy::pedantic, clippy::nursery)]
```

Comment:
:+1: 

split these into one line per warning, so that they can be selectively commented/uncommented